fire/docstrings_test.py

```diff
@@ -257,7 +257,7 @@ def test_numpy_format_multiline_arg_description(self):
                     description='The first parameter.'),
             ArgInfo(name='param2', type='str',
                     description='The second parameter. This has a lot of text, '
-                                'enough to cover two lines.'),
+                                'enough to cover two\nlines.'),
         ],
     )
```

Comment:
These changes to the tests LGTM.

Let's add new tests. One situation to cover is if there are multiple args requiring line merging. There are probably others too.

Reply:
Added 5 new test cases